### PR TITLE
add pyflow

### PR DESCRIPTION
**Pull request by looooo**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by looooo** [comment]
@conda-forge/core it seems like noarch is not working because of the python-script which is copied to /bin. Any ideas how to solve this?

**Comment by looooo** [comment]
as it seems like there is no way to make this noarch python, at least I would like to make only one build for for all python versions. Is there a way to do so?

**Comment by isuruf** [comment]
See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points on how to fix this

File: recipes/pyflow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/wonderworks-software/PyFlow/archive/{{ version }}.tar.gz"
+  sha256: "9a4a481ed9ccfa5295c87bdb32e7c26036351d2857bb9b2e9997c22fc681b856"
+  patches:
+    - setup.patch
```
Comment (isuruf) [inline recipes/pyflow/meta.yaml]: You don't need this, now that you've added the entry_point below

**Comment by looooo** [comment]
thanks @isuruf . So I have to disable the entry-point in the setup.py and enable it for conda-build, right?

**Comment by isuruf** [comment]
> So I have to disable the entry-point in the setup.py and enable it for conda-build, right?

No. You have to enable it for conda-build, but you don't have to disable it in setup.py

**Comment by looooo** [comment]
> 
> 
> > So I have to disable the entry-point in the setup.py and enable it for conda-build, right?
> 
> No. You have to enable it for conda-build, but you don't have to disable it in setup.py

nice, this is conda magic 👍

**Comment by looooo** [comment]
@isuruf using the linux build on windows didn't work. Calling setup.py on windows creates a <script>.exe. But if I use the linux build on windows only a <script> is created (same as linux).

**Comment by isuruf** [comment]
If you re-add `console-scripts` in `meta.yaml`, conda-build will take care of it.

**Comment by looooo** [comment]
sry for the noise, somehow I messed the commit history and the entry-points were removed after the last commit...

File: recipes/pyflow/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+{% set version = "2.0.1" %}
+
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/wonderworks-software/PyFlow/archive/{{ version }}.tar.gz"
+  sha256: "9a4a481ed9ccfa5295c87bdb32e7c26036351d2857bb9b2e9997c22fc681b856"
```
Comment (marcelotrevisani) [inline recipes/pyflow/meta.yaml]: Could you please remove the quotes from ``sha256``?
```suggestion
  sha256: 9a4a481ed9ccfa5295c87bdb32e7c26036351d2857bb9b2e9997c22fc681b856
```

**Pull request merged**